### PR TITLE
Develop training

### PR DESCRIPTION
**Pull request by CLappin**
Adding the training information to the main branch. 
1. YouTube Videos and  Renode webinars added.
2. Updated pages can be seen [here](https://mi-v-ecosystem.github.io/docs/training/)

### CODE REVIEW

File: _docs/11-training.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+excerpt: "Welcome to the Training page."
+last_modified_at: 2018-01-10T11:22:01-05:00
+---
+
+# Training Material
+## Below is a list of training available on Youtube.
```
Comment (kevinscully100) [inline _docs/11-training.md]: "Below is a list of training available on Youtube." -> Training videos on YouTube

File: _docs/11-training.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+layout: splash
+title: "Training"
+permalink: /docs/training/
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+excerpt: "Welcome to the Training page."
```
Comment (kevinscully100) [inline _docs/11-training.md:9]: This "Welcome ..." excerpt on each of the sub-pages may need some attention from KK to provide a focused introduction to the content of the pages. Ciaran, could you note this as an issue to resolve with KK.

File: _docs/11-training.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+last_modified_at: 2018-01-10T11:22:01-05:00
+---
+
+# Training Material
+## Below is a list of training available on Youtube.
+
+* [FPGA](https://www.youtube.com/watch?v=8cXOZ-V31kA&list=PL9B4edd-p2ajpeOI4k4Lr_fwXcRlwO3dv)
```
Comment (kevinscully100) [inline _docs/11-training.md]: Should these YouTube video links be grouped in some way? And the groups ordered? Maybe put the general and hardware related videos first (or last?), then the bare metal s/w related videos, and then the Linux s/w related videos?

Maybe in a later release consider having sub-headings to identify the groups, like for instance:
PolarFire SoC - Linux Applications Training
PolarFire SoC - Bare Metal Applications Training
etc.

File: _docs/11-training.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+* [PolarFire SoC - Github introduction](https://www.youtube.com/watch?v=-e_OniBC5nU&list=PL9B4edd-p2ajvXj0ZflizZI4zgm3Jev_O&index=4)
+* [PolarFire SoC - Bare Metal Interrupts](https://www.youtube.com/watch?v=DOL7GBTKOoA&list=PL9B4edd-p2ahg_MASDPEwOy8spzsL5_f5)
+* [PolarFire SoC - Generating a Design for the PolarFire SoC Icicle Kit with Custom IP](https://www.youtube.com/watch?v=AGmJ7XOjYAM&list=PL9B4edd-p2ahUAL56N_8FuIS3BJFzApEF)
+* [PolarFire SoC - Getting back to factory defaults](https://www.youtube.com/watch?v=dL3u6pYWFvQ&list=PL9B4edd-p2ajvXj0ZflizZI4zgm3Jev_O&index=5)
+* [PolarFire SoC - Factory MSS configuration for the Icicle Kit]( https://www.youtube.com/watch?v=mXp5cNg4qbY&list=PL9B4edd-p2ajvXj0ZflizZI4zgm3Jev_O&index=6)
+* [PolarFire SoC - What is the HSS](https://www.youtube.com/watch?v=UKGVAl3BOog&list=PL9B4edd-p2ajvXj0ZflizZI4zgm3Jev_O&index=8)
+* [PolarFire SoC - MSS Configurator Run through](https://www.youtube.com/watch?v=WrkmaJingRg&list=PL9B4edd-p2ajvXj0ZflizZI4zgm3Jev_O&index=7)
+
+## Webinars
+*  [Renode Webinar Series](https://www.microsemi.com/product-directory/fpga-soc/5210-mi-v-embedded-ecosystem#renode-webinar-series)
```
Comment (kevinscully100) [inline _docs/11-training.md]: Change the display text from "Renode Webinar Series"
to
Getting started with PolarFire SoC - Renode Webinar Series
or
Introduction to PolarFire SoC - Renode Webinar Series

I like the "Getting started ..." option

File: _docs/11-training.md (hunk 2 above)
Comment (CLappin) [reply _docs/11-training.md:9]: Sure, will bring it up with KK offline.

File: _docs/11-training.md (hunk 1 above)
Comment (CLappin) [reply _docs/11-training.md]: Updated in latest commit.

File: _docs/11-training.md (hunk 4 above)
Comment (CLappin) [reply _docs/11-training.md]: Added the Getting Started ... option to the Renode Webinar Series.

File: _docs/11-training.md (hunk 3 above)
Comment (CLappin) [reply _docs/11-training.md]: Yes, I think that is a good idea to group them as you have stated above. Worth doing this in this release?

Comment (kevinscully100) [reply _docs/11-training.md]: I think it is worth doing now - that is reshuffling the list of links. Leave the sub-headings for another time.

Comment (CLappin) [reply _docs/11-training.md]: I have done the reshuffling of the list in the latest commit.

**Pull request merged**